src/main/scala/cromwell/engine/db/slick/LocalJobComponent.scala

```diff
@@ -1,5 +1,6 @@
 package cromwell.engine.db.slick
 
+//TODO: Maybe replace the name of this to just, Job, say?
```

Comment:
One potential issue with all of this is that the DB needs to sometimes track backend-specific information. With backends being pluggable that becomes difficult to do in a codified fashion - it's not insurmountable but will need to think it through in a fashion which doesn't become super inefficient